### PR TITLE
containerd: update to 2.0.1.

### PR DESCRIPTION
**Pull request by tsndqst**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### Local build testing
- I built this PR locally for my native architecture, (x86_64-glibc)
- I built this PR locally for these architectures (if supported. mark crossbuilds):
  - aarch64

### CODE REVIEW

**Comment by tsndqst** [comment]
I need some help with generating the man pages when cross compiling.

File: srcpkgs/containerd/patches/skip_tests.patch
Comment (classabbyamp) [inline srcpkgs/containerd/patches/skip_tests.patch:1]: you can skip tests with `make_check_args="-skip TestName"`

File: srcpkgs/containerd/template [hunk 1]
```diff
@@ -1,12 +1,10 @@
 # Template file for 'containerd'
 pkgname=containerd
-version=1.7.17
-revision=2
+version=2.0.0
+revision=1
 build_style=go
```
Comment (classabbyamp) [inline srcpkgs/containerd/template:5]:
```suggestion
build_style=go
build_helper="qemu"
```

File: srcpkgs/containerd/template [hunk 2]
```diff
@@ -15,16 +13,42 @@ go_build_tags="seccomp apparmor"
...
+	        mips*) export GOARCH=mips;;
+	        riscv64*) export GOARCH=riscv64;;
+	esac
+
+	# Build the man pages on the host machine
+	CGO_CFLAGS=${CFLAGS_host} CC=${GCC_host} CC_target=${GCC_host} \
+		GCC=${GCC_host} GCC_target=${GCC_host} \
+		go build -o bin/gen-manpages  ./cmd/gen-manpages
+	bin/gen-manpages containerd.8 man
+	bin/gen-manpages ctr.8 man
```
Comment (classabbyamp) [inline srcpkgs/containerd/template:50]: you can use `vtargetrun` to run target executables when crosscompiling

File: srcpkgs/containerd/patches/skip_tests.patch
Comment (tsndqst) [reply srcpkgs/containerd/patches/skip_tests.patch:1]: Skipping tests this way doesn't seem to work for this package.

File: srcpkgs/containerd/template (hunk 2 above)
Comment (tsndqst) [reply srcpkgs/containerd/template]: Thanks, `vtargetrun` works.

File: srcpkgs/containerd/patches/skip_tests.patch
Comment (tsndqst) [reply srcpkgs/containerd/patches/skip_tests.patch:1]: Oops, I was overriding `do_check`.  I'll try again with the normal `do_check` function.

**Pull request merged**